src/_includes/how_to/get_connected_pricing_table.html

```diff
@@ -25,9 +25,9 @@
       <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered border-top-0" scope="row">
         Tiers<br />
       </th>
-      <td class="border-dark-blue border-top-0 border-start-0 border-end-0"><em>< 3,000 SIMs</em></td>
-      <td class="border-dark-blue border-top-0 border-start-0 border-end-0"><em>3k - 9,999 SIMs</em></td>
-      <td class="border-dark-blue border-top-0 border-left-0"><em>>9,999 SIMs</em></td>
+      <td class="border-dark-blue border-top-0 border-start-0 border-end-0"><em>Tier 1<br>< 3,000 SIMs</em></td>
+      <td class="border-dark-blue border-top-0 border-start-0 border-end-0"><em>Tier 2<br>3k - 9,999 SIMs</em></td>
+      <td class="border-dark-blue border-top-0 border-left-0"><em>>Tier 3<br>9,999 SIMs</em></td>
```

Comment:
It looks like the greater than sign is in the wrong place

![image](https://github.com/cal-itp/mobility-marketplace/assets/25497886/9a0e3f6d-e99c-4b84-94c7-e018fb3b438a)

```suggestion
      <td class="border-dark-blue border-top-0 border-left-0"><em>Tier 3<br>>9,999 SIMs</em></td>
```

Reply:
ty!!!!